<launch>
    <node pkg="jaka_planner" type="moveit_control" name="moveit_control" output="screen" />
    <param name="desired_force" value="0.0"/>
    <param name="kp" value="0.01"/>
    <param name="kd" value="0.06"/>
    <param name="mass" value="0.003"/>
    <param name="kpZ" value="0.5"/>
    <param name="kdZ" value="0.1"/>

    <param name="interpolation_dist" value="0.8"/>
    <param name="servo_stepnum" value="1.0"/>
    <param name="joint_move_speed" value="50.0"/>
    <param name="linear_move_speed" value="1000.0"/>

    <param name="robot_ip" value="192.168.31.112"/>
    <param name="robot_model" value="s5"/>
	</node>
</launch>

